.vscode/launch.json

```diff
@@ -47,7 +47,7 @@
             "name": "Server/Client",
             "configurations": [
                 "Server",
-                "Client"
+                "Client (Compatibility renderer)"
```

Comment:
these changes slipped through, be sure to .gitignore these or dont push these to a live repo.